### PR TITLE
Onboarding Page backend can now update the User's table with new attributes, made UI changes to Sign Up, LogIn, Home and Onboarding pages

### PR DESCRIPTION
**Pull request by rushanniazi**

### CODE REVIEW

File: travvy-server/src/controllers/AuthenticationController.js [hunk 1]
```diff
@@ -31,8 +31,24 @@ module.exports = {
         //retun the user that was found
         //if there is not a user that exists it will return null
         return user
-    }
+    },
     
+    async onboarding (email, age, gender, instagram_username, preferred_activity_type) {
+        //here we find a user who's email matches an email in the user database
+        const user = await Users.update({
```
Comment (dyampols) [inline travvy-server/src/controllers/AuthenticationController.js:38]: Great to see this working in conjunction with the sign up feature

File: travvy/src/components/Onboarding.vue [hunk 2]
```diff
@@ -29,20 +34,38 @@
...
+    <br>
+    <label>preferred_activity_type?: </label>
+    <select class="info" v-model="preferred_activity_type">
+        <option></option>
+        <option>Family</option>
+        <option>Historical</option>
+        <option>Active</option>
+        <option>Nature</option>
+        <option>Entertainment</option>
+        <option>Food</option>
```
Comment (dyampols) [inline travvy/src/components/Onboarding.vue:46]: This information will need to be taken into account when recommending attractions as a ticket for next sprint

File: travvy/src/components/Onboarding.vue [hunk 3]
```diff
@@ -60,24 +83,50 @@ export default {
...
-    async signup(){
-      console.log(this.name)
+    async onboarding(){
       console.log(this.email)
-      console.log(this.password)
-                
+      console.log(this.age)
+      console.log(this.gender)
+      console.log(this.instagram_username)
+      console.log(this.preferred_activity_type)
```
Comment (dyampols) [inline travvy/src/components/Onboarding.vue:103]: Maybe these logs can be removed for cleaner code

**Review by dyampols — APPROVED** [review]
LGTM - As a todo for next sprint a unit test can be written as well the preferred activities can be taken into account when recommending attractions, great work!

File: travvy/src/components/Onboarding.vue (hunk 3 above)
Comment (rushanniazi) [reply travvy/src/components/Onboarding.vue:103]: Thats a good idea! I added those when i was testing the functionality of the function

File: travvy/src/components/Onboarding.vue (hunk 2 above)
Comment (rushanniazi) [reply travvy/src/components/Onboarding.vue:46]: Yup, next sprint we can add this functionality to our Search feature for the AttractionsList page

File: travvy-server/src/controllers/AuthenticationController.js [hunk 4]
```diff
@@ -31,8 +31,24 @@ module.exports = {
...
+            Age: age,
+            Gender: gender,
+            Instagram_Username: instagram_username,
+            Preferred_Activity_Type: preferred_activity_type,
+           } , {
+            where:{
+                email: email,
+            }
+        })
```
Comment (lkdfount) [inline travvy-server/src/controllers/AuthenticationController.js:47]: Awesome to see you combined the log in and sign up functions

**Comment by lucasungaretti** [comment]
Everything looks good to me! It looks nicer now.

**Pull request merged**